docker-compose.ci.yml

```diff
@@ -5,16 +5,7 @@ services:
       SPRING_LIQUIBASE_ENABLED: "true"
       GIT_DISCOVERY_ACROSS_FILESYSTEM: 1
   frontend:
-    environment:
-      REACT_APP_BASE_URL: https://localhost.simplereport.gov
-      REACT_APP_BACKEND_URL: https://localhost.simplereport.gov/api
-      PUBLIC_URL: /app/
-      REACT_APP_OKTA_ENABLED: "false"
-      REACT_APP_DISABLE_MAINTENANCE_BANNER: "true"
-      GIT_DISCOVERY_ACROSS_FILESYSTEM: 1
-    command: ["./ci-git-fix.sh"]
-  nginx:
```

Comment:
Yesterday we moved the wiremock into this file, you might consider setting the `depends_on: wiremock` in this file and remove it from the base `docker-compose.yml`.

Reply:
good callout, `nginx` dependency on `wiremock` was also removed when we did the move yesterday, we should be good there